frontend/src/components/Layout/Header/ConfirmLogOut/ConfirmLogOut.tsx

```diff
@@ -34,9 +43,15 @@ const ConfirmLogOut: React.FC<IConfirm> = ({ handleLogOut, cancelLogOut, isConfi
         <ButtonCancel variant="mediumOutlined" onClick={cancelLogOut}>
           Cancel
         </ButtonCancel>
-        <ButtonExit variant="mediumContained" onClick={handleLogOut}>
-          Exit
-        </ButtonExit>
+        {status === 'loading' ? (
```

Comment:
We have isLoading prop instead of it!
Replace this condition

Reply:
Replaced